test/Sdk.Generator.Tests/FunctionExecutor/DependentAssemblyTest.cs

```diff
@@ -18,10 +20,10 @@ public DependentAssemblyTest()
             {
                 var abstractionsExtension = Assembly.LoadFrom("Microsoft.Azure.Functions.Worker.Extensions.Abstractions.dll");
                 var httpExtension = Assembly.LoadFrom("Microsoft.Azure.Functions.Worker.Extensions.Http.dll");
-                var hostingExtension = Assembly.LoadFrom("Microsoft.Extensions.Hosting.dll");
-                var diExtension = Assembly.LoadFrom("Microsoft.Extensions.DependencyInjection.dll");
-                var hostingAbExtension = Assembly.LoadFrom("Microsoft.Extensions.Hosting.Abstractions.dll");
-                var diAbExtension = Assembly.LoadFrom("Microsoft.Extensions.DependencyInjection.Abstractions.dll");
+                var hostingExtension = typeof(HostBuilder).Assembly;
```

Comment:
Test cleanup items added to avoid dependency mismatches and errors. Can pull out to separate PR to go in before this one if desired.